app/server/models/transfer_account.py

```diff
@@ -95,14 +95,9 @@ def delete_transfer_account_from_user(self, user: User):
         except (ResourceAlreadyDeletedError, TransferAccountDeletionError) as e:
             raise e
 
-    def get_float_transfer_account(self):
-        for transfer_account in self.organisation.transfer_accounts:
-            if transfer_account.account_type == 'FLOAT':
-                return transfer_account
-
-        float_wallet = TransferAccount.query.filter(TransferAccount.account_type == TransferAccountType.FLOAT).first()
-
-        return float_wallet
+    @property
+    def unrounded_balance(self):
```

Comment:
Had to do this, because when draining randomly generated accounts, I'd get rounding errors when using standard balances